src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -73,6 +73,17 @@ void DnsStreamHandler::start()
             _f_qnames.emplace_back(std::move(qname_ci));
         }
     }
+    if (config_exists("dnstap_msg_type")) {
+        auto type = config_get<std::string>("dnstap_msg_type");
+        try {
+            auto type_pair = _dnstap_map_types.at(type);
+            _f_dnstap_types.set(type_pair.first);
+            _f_dnstap_types.set(type_pair.second);
+            _f_enabled.set(Filters::DnstapMsgType);
+        } catch (const std::exception &e) {
+            throw ConfigException(fmt::format("dnstap_msg_type error {}", e.what()));
```

Comment:
let's go ahead and dump the keys in _dnstap_map_types in this message so the user knows which they can set

Reply:
ok